documentation/staging/content/userguide/managing-domains/model-in-image/common-mounts.md

```diff
@@ -0,0 +1,678 @@
++++
+title = "Common mounts"
+date = 2019-02-23T16:45:16-05:00
+weight = 25
+pre = "<b> </b>"
+description = "Common mounts are an alternative approach for including Model in Image model files, application archive files, WebLogic Deploying Tooling installation files, or other types of files, in your pods."
++++
+
+### Contents
+
+ - [Introduction](#introduction)
+ - [References](#references)
+ - [Configuration](#configuration)
+   - [Common mount images](#common-mount-images)
+   - [Common mount volumes and paths](#common-mount-volumes-and-paths)
+   - [Model in Image paths](#model-in-image-paths)
+ - [Sample](#sample)
+    - [Step 1: Prerequisites](#step-1-prerequisites)
+    - [Step 2: Create the common mounts image](#step-2-create-the-common-mounts-image)
+    - [Step 3: Prepare and apply the domain resource](#step-3-prepare-and-apply-the-domain-resource)
+    - [Step 4: Invoke the web application](#step-4-invoke-the-web-application)
+
+### Introduction
+
+{{% notice warning %}}
+The common mounts feature is a work in progress and is currently unsupported.
+Its configuration or behavior may change between releases and it is disabled by default.
+If you want to enable this feature, then set your operator's `"featureGates"`
+Helm configuration attribute to include `"CommonMounts=true"`.
+The `"featureGates"` attribute acknowledges use of an unsupported feature,
+will not be required after common mounts is fully supported,
+defaults to being unset, and accepts a comma-separated list.
+{{% /notice %}}
+
+Common mounts are an alternative approach for including Model in Image model files,
+application archive files, WebLogic Deploying Tooling installation files,
+or other types of files, in your pods.
+This feature eliminates the need to provide these files in the image specified
+in `domain.spec.image`.
+
+Instead:
+
+- The domain resource's `domain.spec.image` directly references a base image
+  that needs to include only a WebLogic installation and a Java installation.
+- The domain resource's common mount related fields reference one or
+  more smaller images that contain the desired Model in Image files.
+- The domain resource's `domain.spec.configuration.model.wdtInstallHome`
+  and `domain.spec.configuration.model.modelHome` fields are set to
+  reference a directory that contains the files from the smaller images.
+
+The advantages of common mounts for Model In Image domains are:
+
+- Use or patch a WebLogic installation image without needing to include a WDT installation,
+  application archive, or model artifacts within the image.
+- Share one large WebLogic installation image with multiple different model
```

Comment:
Let's not highlight that WebLogic images are particularly large ;). Perhaps remove the word "large" and "smaller" becomes "specific".

Reply:
Fixed.